packages/@aws-cdk/core/test/test.construct.ts

```diff
@@ -381,11 +381,11 @@ export = {
 
     class LockableConstruct extends Construct {
       public lockMe() {
-        (this.node as any)._lock();
+        (this.node._actualNode as any)._lock();
```

Comment:
Why not just implement `_lock` on the wrapping `ConstructNode`?

Reply:
This is just for the test. I don’t know that we need to pollute the api (even if it’s internal). What do you think?

Reply:
Yeah you're right. Fugettaboutit.